### ISSUE TITLE
[Feature?] gatsby-theme-tsdx? or docusaurus template

### ISSUE THREAD

**Issue by sw-yx** [palmerhq/tsdx#176]
### Desired Behavior

we have #78 but we need a simple hosting solution for people to navigate to their error codes.

### Suggested Solution

write a simple gatsby theme for this that can be tacked on to any docsite. or docusaurus template altho this is less customizable.

### Who does this impact? Who is this for?

users of libraries maintained with tsdx

### Describe alternatives you've considered

n/a

**Comment by jaredpalmer** [comment]
Yep. I can see some kind of `gatsby-plugin-tsdx-errors` as well.

**Comment by sw-yx** [comment]
(this is one of my todos but i've been busy. would help to have a simple design as i quite lack in that department)
